frameworks/JavaScript/nodejs/app.js

```diff
@@ -0,0 +1,16 @@
+var cluster = require('cluster');
+var numCPUs = require('os').cpus().length;
+
+if (cluster.isMaster) {
+  // Fork workers.
+  for (var i = 0; i < numCPUs; i++) {
+    cluster.fork();
+  }
+
+  cluster.on('exit', function (worker, code, signal) {
+    console.log('worker ' + worker.pid + ' died');
```

Comment:
Not a single worker should die during the benchmark. If this is called, it means the app is missing error handling somewhere. I recommend shutting everything down if this callback is called and stopping the test.